docs/archiver/archiver-guide-sepolia-sync-gamma.md

```diff
@@ -0,0 +1,233 @@
+t=2025-09-16T13:15:32+0200 lvl=info msg="generated attributes in payload queue" txs=1 timestamp=1757986826
+t=2025-09-16T13:15:32+0200 lvl=info msg="Record safe head" l2=0x4f03259baa62710548cc25aaddc477a93bf081627df5d705a0fd232529c5530e:2276964 l1=0x7752fcf783c6bc3d6c55ff676ba3c579ca99f0f0dc84d185f900c36f7e535a32:9213344
+t=2025-09-16T13:15:32+0200 lvl=info msg="Inserted new L2 unsafe block" hash=0x61c9fc0463ffde42a3f2919d1a2cbde63e139014bddb88a2a2d33e20f5f45c8e number=2276965 state_root=0xc1f3e6231766bbd63733729eb5ea5d436825c87c06e27486c68f5ca14e35ed08 timestamp=1757986826 parent=0x4f03259baa62710548cc25aaddc477a93bf081627df5d705a0fd232529c5530e prev_randao=0x48fd16fb17500bcbe13146690acd6eaa3e638329521a1d58bfec51192f9deba0 fee_recipient=0x4200000000000000000000000000000000000011 txs=1 build_time=1.657ms insert_time=2.112ms total_time=3.769ms mgas=0.048637 mgasps=12.90153668795746
+```
+
+**op-geth logs:**
+```log
+INFO [09-16|13:11:34.578] Imported new potential chain segment     number=2,275,756 hash=1900fd..127eea blocks=1 txs=1 mgas=0.049 elapsed="947.009µs" mgasps=51.359   age=10h11m26s snapdiffs=2.85MiB    triedirty=10.73MiB
+INFO [09-16|13:11:34.579] Chain head was updated                   number=2,275,756 hash=1900fd..127eea root=a9eb8c..feea18 elapsed="668.261µs" age=10h11m26s
+INFO [09-16|13:11:34.583] Starting work on payload                 id=0x0301f10e4c14fc3c
+INFO [09-16|13:11:34.585] Imported new potential chain segment     number=2,275,757 hash=b63896..98e6c0 blocks=1 txs=1 mgas=0.049 elapsed=1.063ms     mgasps=45.734   age=10h11m24s snapdiffs=2.85MiB    triedirty=10.73MiB
+INFO [09-16|13:11:34.586] Chain head was updated                   number=2,275,757 hash=b63896..98e6c0 root=a94226..d705f7 elapsed="845.15µs"  age=10h11m24s
+```
+
+**es-node logs:**
+```log
+INFO [09-16|09:26:23.911] Blob archiver API request                url="/eth/v1/beacon/blob_sidecars/8514581?indices=3"
+INFO [09-16|09:26:24.367] Blob archiver API request handled        took(s)=0.456
+INFO [09-16|10:27:09.649] Blob archiver API request                url="/eth/v1/beacon/blob_sidecars/8518431?indices=1"
+INFO [09-16|10:27:10.063] Blob archiver API request handled        took(s)=0.414
+INFO [09-16|12:31:19.786] Blob archiver API request                url="/eth/v1/beacon/blob_sidecars/8522289?indices=5"
+INFO [09-16|12:31:20.362] Blob archiver API request handled        took(s)=0.576
+```
+
+From the logs, we can see that the op-node keeps converting batches into payload attributes, calling the op-geth to work on the payload to convert it into blocks, and inserting the blocks as the chain head.
+
+Additionally, you may notice the age of derived blocks is over 12 hours, and blob archiver API requests were handled by the es-node, meaning the blobs were retrieved from EthStorage since they were pruned by the L1 Beacon client (mocked).
```

Comment:
It’s still unclear under what conditions we can reproduce this and make op-node fetch the blob from the archive API instead of the mock beacon API.